content/courses/program-optimization/rust-macros.md

```diff
@@ -451,12 +454,12 @@ Finally, the `expanded` variable is converted into a `TokenStream` using the
 `into()` method.
 
 ```rust
-use proc_macro::{self, TokenStream};
+use proc_macro::TokenStream;
 use quote::quote;
-use syn::{parse_macro_input, DeriveInput, FieldsNamed};
+use syn::{parse_macro_input, DeriveInput, Data, Fields};
```

Comment:
`syn::Data` and `syn::Fields` is being imported, but the import is not being properly used in the `describe_struct` associated function